zproject/urls.py

```diff
@@ -565,12 +566,12 @@
     # Registration views, require a confirmation ID.
     path("accounts/home/", accounts_home),
     path(
-        "accounts/send_confirm/<email>",
-        TemplateView.as_view(template_name="zerver/accounts_send_confirm.html"),
+        "accounts/send_confirm",
+        signup_send_confirm,
         name="signup_send_confirm",
     ),
     path(
-        "accounts/new/send_confirm/<email>",
+        "accounts/new/send_confirm",
```

Comment:
It looks like basically every other URL under `accounts/` has a trailing slash in the URL; let's use that here too?

Reply:
Okay.